app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -399,4 +399,7 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!--  ReadingTextSizeFragment-->
+  <dimen name="reading_text_size_fragment_padding_bottom">132dp</dimen>
+
 </resources>
```

Comment:
As `reading_text_size_fragment_padding_bottom` has the same value as in `dimens.xml` so you can delete extra declaration from here.